### PR TITLE
Add appveyor and fix upload config 

### LINKED ISSUE #9: [TEST]Specialized tests needed under Windows
As plasma store does not support Windows currently, cases for web sessions on a mock server are needed to make sure Mars client works properly under Windows.

### PR DESCRIPTION
**Pull request by NJUPole**
## What do these changes do?

Add appveyor config, as plasma doesn't support in Windows, we skip all the cases under mars/worker and `test_main` in mars/scheduler. 

Add `TestWithMockServer` in test_api to make sure Mars client works properly under Windows.

`test_gevent_pool` is now skipped, it will be fixed.

Fix travis upload config.

## Related issue number

resolve #9 
#45

### CODE REVIEW

File: mars/scheduler/tests/test_assigner.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def get_reply(self):
 class Test(unittest.TestCase):
 
     def testAssignerActor(self):
-        with create_actor_pool(backend='gevent') as pool:
+        with create_actor_pool(n_process=1, backend='gevent') as pool:
```
Comment (chineking) [inline mars/scheduler/tests/test_assigner.py]: @wjsi Do we have any process ability to test on the AssignerActor? Is it fine to create only a process?

Comment (wjsi) [reply mars/scheduler/tests/test_assigner.py]: AssignerActor does not rely on processes

Comment (chineking) [reply mars/scheduler/tests/test_assigner.py]: good, resolved

File: mars/appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      PYTHON_ARCH: "64"
+
+    - PYTHON: "C:\\Python37-x64"
+      PYTHON_VERSION: "3.7.0"
+      PYTHON_ARCH: "64"
+
+install:
+  # Install Python (from the official .msi of http://python.org) and pip when
+  # not already installed.
+  - ps: if (-not(Test-Path($env:PYTHON))) { & bin\appveyor_install.ps1 }
```
Comment (wjsi) [inline mars/appveyor.yml]: Delete this when appveyor_install.ps1 does not exist

**Pull request merged**